lists/global.csv

```diff
@@ -1465,3 +1465,7 @@ https://clubhouse.pubnub.com/,GRP,Social Networking,2021-04-04,Chelchela,
 https://brave.com/,HACK,Hacking Tools,2021-06-24,Chelchela,
 https://search.brave.com/,SRCH,Search Engines,2021-06-24,Chelchela,
 https://startpage.com/,SRCH,Search Engines,2021-06-24,Chelchela,
+https://s.to/,FILE,CUII-Recommendation (cuii.info),2021-07-30,mhombach
+http://canna.to/,FILE,CUII-Recommendation (cuii.info),2021-07-30,mhombach
+https://nsw2u.com/,FILE,CUII-Recommendation (cuii.info),2021-07-30,mhombach
+https://newalbumreleases.net/,FILE,CUII-Recommendation (cuii.info),2021-07-30,mhombach
```

Comment:
There is an issue with the format of these URLs in the test list. The third column should be the description of the category code, in this case it should read `File-sharing` instead of `CUII-Recommendation (cuii.info)`

Reply:
> 

Hey, sorry I didn't get the notification for your reply here for some reason.
I can totally change that, my idea was just, that those links would be blocked by a specific company/organization and so we could group them together... Do you think there some way lateron to implement some "notes/comments" into a list entry, which could be displayed in the app?
But for this PR I can just change that column, thanks for the response and feedback!

Reply:
@hellais Fixed everything, can you have a look again? =)